cpp/ql/src/Security/CWE/CWE-120/VeryLikelyOverrunWrite.ql

```diff
@@ -0,0 +1,34 @@
+/**
+ * @name Likely overrunning write based on non-trivial analysis of value ranges
+ * @description Buffer write operations that do not control the length
+ *              of data written may overflow
+ * @kind problem
+ * @problem.severity error
+ * @security-severity 9.3
+ * @precision high
+ * @id cpp/very-likely-overrunning-write
+ * @tags reliability
+ *       security
+ *       external/cwe/cwe-120
+ *       external/cwe/cwe-787
+ *       external/cwe/cwe-805
+ */
+
+import semmle.code.cpp.security.BufferWrite
+import semmle.code.cpp.commons.Alloc
+
+/*
+ * See CWE-120/UnboundedWrite.ql for a summary of CWE-120 alert cases.
```

Comment:
The table in `CWE-120/UnboundedWrite.ql` should have a reference to the new query actually, alongside `OverrunWrite.ql` and `OverrunWriteFloat.ql`.